src/Controller/EventsController.php

```diff
@@ -14,10 +14,10 @@
 use Joindin\Api\Model\TalkMapper;
 use Joindin\Api\Model\TrackMapper;
 use Joindin\Api\Model\UserMapper;
+use Joindin\Api\Request;
 use Joindin\Api\Service\EventApprovedEmailService;
 use Joindin\Api\Service\EventSubmissionEmailService;
 use PDO;
-use Joindin\Api\Request;
```

Comment:
Is this change related to increasing the `phpstan` level?

Reply:
It is - one of the things it calls out is unused imports.

Reply:
Hmm. Now I'm not so sure. I was running PHPStan, Phan and Psalm. I'm going to add it back.